### PR TITLE
reformat criteria_finder()

### PR DESCRIPTION
**Pull request by 0atman**
Quick bit of readability fixes.

### CODE REVIEW

**Comment by danjoneslf** [comment]
Should we get this merged or refactor the other bit out of the DatabaseHandle class as part of the same PR?

**Comment by 0atman** [comment]
Let's merge it - I'm a fan of small, granular PRs. Can we test it?

File: chalice/chalicelib/admin.py [hunk 1]
```diff
@@ -185,6 +162,27 @@ def database_list_models(event, context):
...
+
+        members = inspect.getmembers(
+            importlib.import_module(parent_module_import_path), inspect.isclass
+        )
+        for name, cls in members:
+            active_criteria.add(f"{parent_module_import_path}.{name}")
+
+    return active_criteria
+
+
```
Comment (0atman) [inline chalice/chalicelib/admin.py]: note to self: Make refactors in a separate commit to moves.

**Comment by 0atman** [comment]
[https://github.com/alphagov/csw-backend/pull/132](https://trello.com/c/7L1McPjX/4-https-githubcom-alphagov-csw-backend-pull-132)

**Comment by 0atman** [comment]
[https://github.com/alphagov/csw-backend/pull/132](https://trello.com/c/7L1McPjX/4-https-githubcom-alphagov-csw-backend-pull-132)
